cross/go2rtc/digests

```diff
@@ -0,0 +1,3 @@
+go2rtc-1.8.0.tar.gz SHA1 f4957e44099e6782d3df7b847623bea8bfd68537
+go2rtc-1.8.0.tar.gz SHA256 6ab129c2435fd2ae699e623d51d83bef4eb0220e7f60083e10f4d3992e2e8532
+go2rtc-1.8.0.tar.gz MD5 1368d51872e9c7d49b23966280c65876ч
```

Comment:
If you create this file with "make clean && make digests" in `cross/go2rtc` it will not miss the filnal line break.

Reply:
thx. it was a typo